packages/@o3r/git-tools/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@o3r/git-tools",
```

Comment:
The name seems quite generic and not really matching the readme description.
Maybe we can give a name to be more specific and indicate that it expose new version compute functions only.
I am not sure that future git tooling will not bring dependencies and then require their own package as well.